wolfcrypt/src/port/nxp/dcp_port.c

```diff
@@ -49,6 +51,15 @@
 #include "fsl_debug_console.h"
 #include "fsl_dcp.h"
 
+#ifdef USE_FAST_MATH
+    #include <wolfssl/wolfcrypt/tfm.h>
+#elif defined WOLFSSL_SP_MATH
+    #include <wolfssl/wolfcrypt/sp_int.h>
+#else
+    #include <wolfssl/wolfcrypt/integer.h>
+#endif
```

Comment:
```suggestion
#include <wolfssl/wolfcrypt/integer.h>
```

Reply:
Thanks, I've removed the includes entirely now.